test/unittests/test_instructions.cpp

```diff
@@ -15,7 +15,7 @@ TEST(instructions, homestead_hard_fork)
 
     EXPECT_EQ(f[OP_DELEGATECALL].gas_cost, -1);
     EXPECT_EQ(h[OP_DELEGATECALL].gas_cost, 40);
-    EXPECT_EQ(fn[OP_DELEGATECALL], nullptr);
+    EXPECT_TRUE(fn[OP_DELEGATECALL] == nullptr);
```

Comment:
Why are these needed?

Reply:
Printing of `nullptr` does not compile in MSVC because of the GTest issue.